hack/robot-e2e/ansible.cfg

```diff
@@ -0,0 +1,7 @@
+[defaults]
+inventory = ${PWD}/inventory.yml
+forks = 10
```

Comment:
I think we can use the default and remove this. We don't have many hosts to manage.